service/internal/builders/processor_test.go

```diff
@@ -24,12 +24,12 @@ func TestProcessorBuilder(t *testing.T) {
 	defaultCfg := struct{}{}
 	factories, err := processor.MakeFactoryMap([]processor.Factory{
 		processor.NewFactory(component.MustNewType("err"), nil),
-		processor.NewFactory(
+		processorprofiles.NewFactory(
 			component.MustNewType("all"),
 			func() component.Config { return &defaultCfg },
-			processor.WithTraces(createProcessorTraces, component.StabilityLevelDevelopment),
-			processor.WithMetrics(createProcessorMetrics, component.StabilityLevelAlpha),
-			processor.WithLogs(createProcessorLogs, component.StabilityLevelDeprecated),
+			processorprofiles.WithTraces(createProcessorTraces, component.StabilityLevelDevelopment),
+			processorprofiles.WithMetrics(createProcessorMetrics, component.StabilityLevelAlpha),
+			processorprofiles.WithLogs(createProcessorLogs, component.StabilityLevelDeprecated),
```

Comment:
I think we should rename `processorprofiles` to something like `expprocessor` without waiting for another new signal

Reply:
Can do that later, since is experimental.